projects/plugins/migration/tests/e2e/package.json

```diff
@@ -0,0 +1,51 @@
+{
+	"name": "jetpack-starter-e2e-tests",
```

Comment:
Do you plan on adding E2E tests to the project? If so, you might want to update the references to the starter-plugin. It's not a blocker or anything, just FYI.

Reply:
I think it makes sense for us to update it anyway, thanks for catching this @brbrr !